locations/spiders/pizza_hut_tr.py

```diff
@@ -0,0 +1,31 @@
+import scrapy
+from scrapy.http import JsonRequest
+
+from locations.categories import Categories, apply_category
+from locations.dict_parser import DictParser
+
+
+class PizzaHutTRSpider(scrapy.Spider):
+    name = "pizza_hut_tr"
+    item_attributes = {"brand": "Pizza Hut", "brand_wikidata": "Q191615"}
```

Comment:
Maybe it makes sense to share these attributes across all Pizza Hut spiders, similar to Burger King spiders.